paywall/unlock.js.webpack.config.js

```diff
@@ -0,0 +1,22 @@
+var path = require('path')
+const webpack = require('webpack')
+
+module.exports = env => {
+  return {
+    mode: 'production',
```

Comment:
Could we derive that from `UNLOCK_ENV` instead? Using `development` could make things easier to debug locally?

Reply:
because it generates a source map, this shouldn't be an issue for debugging in chrome. Did you mean just to check which files are included in the bundle?